plugins/node/opentelemetry-instrumentation-pg/src/pg.ts

```diff
@@ -0,0 +1,302 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  isWrapped,
+  InstrumentationBase,
+  InstrumentationConfig,
+  InstrumentationNodeModuleDefinition,
+} from '@opentelemetry/instrumentation';
+
+import {
+  context,
+  getSpan,
+  Span,
+  SpanKind,
+  StatusCode,
+} from '@opentelemetry/api';
+import * as pgTypes from 'pg';
```

Comment:
```suggestion
import type * as pgTypes from 'pg';
```

Reply:
check the code of pg plugin, this is moving plugin to instrumentation not to create a new package. I'm not changing anything besides moving this to instrumentation.